playwright/src/test/java/com/microsoft/playwright/TestPlaywrightCustomFixtures.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.playwright;
+
+import com.microsoft.playwright.junit.Options;
+import com.microsoft.playwright.junit.OptionsFactory;
+import com.microsoft.playwright.junit.UsePlaywright;
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@UsePlaywright(TestPlaywrightCustomFixtures.CustomOptions.class)
+public class TestPlaywrightCustomFixtures {
+  public static class CustomOptions implements OptionsFactory {
+    @Override
+    public Options getOptions() {
+      return new Options().setBaseUrl("https://bing.com").setBrowserName("firefox");
```

Comment:
Let's use a local url, so that we don't depend on external servers in the tests. Something like `server.EMPTY_PAGE` from other tests should work.